addons/sale/wizard/sale_make_invoice_advance.py

```diff
@@ -311,9 +311,19 @@ def _prepare_down_payment_lines_values(self, order):
         if self.advance_payment_method == 'percentage':
             percentage = self.amount / 100
         else:
-            percentage = self.fixed_amount / order.amount_total if order.amount_total else 1
-
-        order_lines = order.order_line.filtered(lambda l: not l.display_type and not l.is_downpayment)
+            invoice_lines = order.order_line.invoice_lines.filtered(
+                lambda aml: aml.invoice_status == 'invoiced'
+            )
```

Comment:
Whoops, seems like `invoice_status` is a field for order lines, invoice lines have `parent_state` instead:

```suggestion
            invoice_lines = order.order_line.invoice_lines.filtered(
                lambda aml: aml.parent_state != 'cancel'
            )
```